### DISCUSSION TITLE
realy poor job on chart and chechout

### DISCUSSION THREAD

**Discussion by MartinBjerg** [woocommerce/woocommerce#51604] [Checkout Flow]
It is really poor that it is not possible to translate the text on the cart and checkout page. When English and other languages are mixed together, it comes across as very untrustworthy, and you lose customers because of it. Please make the text in the blocks editable without having to go into PO and HTML and all sorts of strange places; it should be easy and straightforward.

Regards Martin

**Comment by senadir** [comment]
Hey! I wouldn’t classify that as “realy poor job” but still. Checkout is fully translatable, like the old Checkout, and partially Editable, unlike the old Checkout which wasn’t editable at all.

What kind of strings do you want to edit? chances are you can do that from the editor or via “Say What” or LocoTranslate plugin.

**Comment by MartinBjerg** [comment]
Hello Seghir

neither say what, loco translate, PO, or anything I try will translate to
Danish, I have tried everything, nothing works, that is why I think it is
poorly made, and I wonder why it can't be done so that you can edit
directly in the box.

The term in question is "Collection from" both in the shopping cart and at
checkout. It only appears when local pickup is selected; otherwise, it
changes when you choose "ship" to the correct phrase in Danish.

Best regards, Martin.

Den man. 23. sep. 2024 kl. 14.51 skrev Seghir Nadir <
***@***.***>:

**Comment by senadir** [comment]
I agree, Danish is sadly not 100% translated yet (sitting at 82%)

https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/da/default/?filters%5Bterm%5D=collection+from&filters%5Bterm_scope%5D=scope_any&filters%5Bstatus%5D=untranslated&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc

I have not tried Loco Translate or PO Editor for some time now, but you should use something that supports JSON strings, not just POT strings.

This is "Say What" plugin:

Translation:
<img width="439" alt="Screenshot 2024-09-23 at 5 43 28 PM" src="https://github.com/user-attachments/assets/8cd5cb56-6f7f-49a6-9169-ce93efc55162">

Before:
<img width="381" alt="image" src="https://github.com/user-attachments/assets/c8fbbf93-ff52-46f2-a3e8-5bd7166b0116">


After:
<img width="347" alt="image" src="https://github.com/user-attachments/assets/96ea66f1-9f7a-4d3f-9be9-bfac733c7f24">

**Comment by MartinBjerg** [comment]
Hello

Ok I did not insert %s that could be the reason for it not to work, Il try
that next time Im online.

Thank for you help and support

man. 23. sep. 2024 17.45 skrev Seghir Nadir ***@***.***>:
